articles/introduction-to-random-forest-in-ml/index.md

```diff
@@ -0,0 +1,99 @@
+
+Our first example can still be used to explain how random forests work. Instead of having a single decision tree, the rain forest will have many decision trees. Let's assume we have only four decision trees. In this case, the training data comprising the phone's observations and features will be divided into four root nodes. 
+
+The root nodes could represent four features that could influence the customer's choice (price, internal storage, camera, and RAM). The random forest will split the nodes by selecting features randomly. The final prediction will be selected based on the outcome of the four trees. The outcome chosen by most decision trees will be the final choice. If three trees predict *buying*, and one tree predicts *not buying*, then the final prediction will be *buying*. In this case, it is predicted that the customer will buy the phone.
+
+### Classification in Random Forest
+Classification in random forests employs an ensemble methodology to attain the outcome. The training data is fed to train various decision trees. This dataset consists of observations and features that will be selected randomly during the splitting of nodes.
+
+A rain forest system relies on various decision trees. Every decision tree consists of a root node, decision nodes, and leaf nodes. The leaf node of each tree is the final output produced by that specific decision tree. The selection of the final output follows the majority-voting system. In this case, the output chosen by the majority of the decision trees becomes the final output of the rain forest system. The diagram below shows a simple random forest classifier. 
+
+![Random Forest Classifier](/engineering-education/introduction-to-random-forest-in-machine-learning/random-forest-classifier.png)
+
+[Image Source: Medium](https://miro.medium.com/max/5752/1*5dq_1hnqkboZTcKFfwbO9A.png)
+
+Let's take an example of a training dataset consisting of various fruits such as bananas, apples, pineapples, and mangoes. The random forest classifier divides this dataset into subsets. These subsets are given to every decision tree in the random forest system. each decision tree produces its specific output. For example, the prediction for trees 1 and 2 is *apple*. Another decision tree (n) has predicted *banana* as the outcome. The random forest classifier collects the majority voting to provide the final prediction. The majority of the decision trees have chosen *apple* as their prediction. This makes the classifier to choose *apple* as the final prediction. 
+
+![Example of Random Forest Classifier](/engineering-education/introduction-to-random-forest-in-machine-learning/example-of-random-forest-classifier.png)
+
+[Image Source: Javatpoint](https://static.javatpoint.com/tutorial/machine-learning/images/random-forest-algorithm2.png)
+
+### Applications of Random Forest

```

Comment:
Also, explain how a random forest can be used as a regression model. Since it is a collection of various decision trees, how does one arrive at contiguous outputs? Include this section before the applications section.